.gog/feature.json

```diff
@@ -0,0 +1 @@
+{"jira":"BENS-0001","comment":"Update Go Download URL","test_count":0}
```

Comment:
what does this mean?

Reply:
Oh sorry normally I use a CLI tool to manage my project feature branches. I can remove that since your project uses PR to merge changes into base.